.circleci/config.yml

```diff
@@ -1,9 +1,32 @@
 version: 2
 
+defaults: &defaults
+  shell: /bin/bash --login
+  docker:
+    - image: circleci/node:10.16.0-stretch
+
 jobs:
-  build:
-    docker:
-      - image: circleci/node:10.16.0-stretch
+
+  publish:
+    <<: *defaults
     steps:
       - checkout
-      - run: echo "simple build executed"
\ No newline at end of file
+      - run:
+          name: Set NPMJS Registry
+          command: npm config set //registry.npmjs.org/:_authToken ${NPM_TOKEN}
```

Comment:
Before we merge, we should add the CircleCI project to specify this environment variable and another Github token and deploy key.